submissions/sprint_a/Pipfile

```diff
@@ -0,0 +1,14 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[dev-packages]
+pytest = "*"
+
+[packages]
+flask = "*"
+python-dotenv = "*"
+
+[requires]
+python_version = "2.7"
```

Comment:
Flask supports python 2.7 and >= 3.4. I recommend using 3.6 and above.
http://flask.pocoo.org/docs/1.0/installation/